osu.Game/Skinning/LegacySkinDecoder.cs

```diff
@@ -32,6 +32,14 @@ protected override void ParseLine(DefaultSkinConfiguration skin, Section section
                             case @"Author":
                                 skin.SkinInfo.Creator = pair.Value;
                                 return;
+
+                            case @"Version":
+                                if (pair.Value == "latest" || pair.Value == "User")
```

Comment:
Please re-read the wiki. "User" is *not* a valid version. It would be based off the skin's name.

Reply:
I'm gonna support that later actually, seems a bit complicated to implement...